blazar_dashboard/content/leases/workflows.py

```diff
@@ -357,16 +491,25 @@ class CreateLease(workflows.Workflow):
                         'has been submitted.')
     failure_message = _('Unable to create the lease named "%s".')
     success_url = reverse_lazy('horizon:project:leases:index')
-    multipart = True
+    wizard = True
```

Comment:
What's the difference between multipart and wizard?

Reply:
multipart is not necessary here. It's used for "multipart/form_data" for POST requests when uploading files. 
wizard is the “prev” and “next” buttons and validation after every step